host/src/main.rs

```diff
@@ -477,12 +481,18 @@ async fn main() -> anyhow::Result<()> {
         wasi.arg(arg);
     }
 
+    let c = native_tls::TlsConnector::builder().build()?;
+        // needed for self signed
+        // do we need a way to pass configuration values to host from client? 
+        // .danger_accept_invalid_hostnames(true)
```

Comment:
How about we add a CLI option to the `Options` struct above that toggles these settings so we don't have to keep editing the file to (un)comment these lines?